pages/dkp/konvoy/2.1/storage/static-local-volume/index.md

```diff
@@ -0,0 +1,128 @@
+---
+layout: layout.pug
+navigationTitle: Provision a static local volume
+title: Provision a static local volume
+menuWeight: 15
+excerpt: Learn how to provision a static local volume for a Konvoy cluster
+beta: false
+enterprise: false
+---
+
+<!-- markdownlint-disable MD004 MD007 MD025 MD030 -->
+
+The `localvolumeprovisioner` addon uses the [local volume static provisioner][localstorage] to manage persistent volumes for pre-allocated disks.
+It does this by watching the `/mnt/disks` folder on each host and creating persistent volumes in the `localvolumeprovisioner` storage class for each disk that is discovered in this folder.
+
+- Persistent volumes with a 'Filesystem' volume-mode are discovered if they are mounted under `/mnt/disks`.
+
+- Persistent volumes with a 'Block' volume-mode are discovered if a symbolic link to the block device is created in `/mnt/disks`.
+
+# Before you begin
+
+Before starting this tutorial, you should verify the following:
+
+- You have access to a Linux, macOS, or Windows computer with a supported operating system version.
+
+- You have a provisioned `konvoy` cluster that is configured to use the `localvolumeprovisioner` addon, but have not any addons to the cluster.
+
+  For this tutorial, you **should not deploy** using all of the default settings as described in the [Quick start][quickstart].
+
+  This distinction between provisioning and deployment is important because some addons might depend on the storage class provided by the `localvolumeprovisioner` addon and might fail to start if it isn't configured yet.
```

Comment:
```suggestion
  This distinction between provisioning and deployment is important because some addons depend on the storage class provided by the `localvolumeprovisioner` addon and can fail to start if it is not configured yet.
```